src/main/java/net/minecraftforge/unification/UnificationForm.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.unification;
+
+/**
+ * An {@link UnificationForm} represents what a {@link Unified} is shaped as.
+ * Examples include ingots, blocks, and ores. See {@link UnificationConstants} for many predefined {@link UnificationForm}s.
+ * Create your own with {@link UnificationManager#getForm(String)}.
+ */
+public final class UnificationForm
```

Comment:
How about injecting with annotations for forms, materials, and unifieds?

Reply:
I am not really a fan of injecting with annotations, it makes static analysis tools confused. 
It could be added on afterward if someone really wants it though.

Reply:
What happens regarding @ObjectHolder with Unified instances? Will they be populated correctly, or will they just get the specific Item with the matching ResourceLocation?

Reply:
@Alpvax they won't get injected since they are not in forge registries.